_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+Title: An update on relicensing the MDAnalysis library (towards LGPLv2.1+)
+
+> ⚠️ **Disclaimer**
+> The MDAnalysis core team members are not
+> lawyers. As such the information provided here __does not, and is not
+> intended to constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing; it
+> simply aims to inform MDAnalysis developers and users on certain
+> aspects of the MDAnalysis relicensing efforts and how we think it
+> may impact them going forward.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Institute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+## Short description
+
+This blog post provides an update on MDAnalysis’ proposed license change. Specifically, we outline our
+decision to opt for a [GNU Lesser General Public License v2.1+ (LGPL v2.1+)][LGPL v2.1] instead of
+the [previously communicated]() [GNU Lesser General Public License v3+ (LGPL v3+)][LGPL v3].
+
+We also detail some [interim changes](#why-the-interim-gpl-v3-and-lgpl-v3-package-licenses-the-issue-with-apache-20)
+to the MDAnalysis [package license](#package-license), alongside some
+[upcoming actions for library contributors](#package-license).
+
+## Overview
+
+In November of 2022 the MDAnalysis team [announced a proposal to change the license of its core library](https://www.mdanalysis.org/2022/11/07/relicensing/#relicensing-mdanalysis)
+from the [GNU General Public License v2+][GPL v2] to the [GNU Lesser General Public License v3+][LGPL v3].
+The full rationale for this change can be found [here](https://www.mdanalysis.org/2022/11/07/relicensing/#rationale-for-license-change)
+and mainly centered around three main arguments:
+
+1. Allowing downstream codes to freely import or link to the MDAnalysis library with minimal worry about the license they choose to use.
+2. Allowing downstream codes to use and subclass any MDAnalysis components under its application programming interface
+   (classes, methods, and data objects), without incurring copyleft and incurring the license choice of the downstream code.
+3. Ensuring that codes that either directly copy or extend the MDAnalysis library’s source code would fall under the copyleft
+   license requirements of the MDAnalysis library license. Doing so would ensure that eventual license would more closely match the
+   original intent of the historical [GPL v2+](GPL v2) contributions which were kindly donated to MDAnalysis over the years.
+
+It is our view that a weak copyleft license, such as the [GNU Lesser General Public License][LGPL v3] would best match these requirements.
```

Comment:
It was

Reply:
Isn't still "is"? I.e. weak copyleft is still good, but LGPLv3 is just not the best?